docs/alerting/monitors.md

```diff
@@ -41,6 +41,39 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
    This information is stored in plain text in the Elasticsearch cluster. We will improve this design in the future, but for now, the encoded credentials (which are neither encrypted nor hashed) might be visible to other Elasticsearch users.
 
+###  Email as a destination
+To send or receive an alert notification as an email, first select **Email** as the destination **Type** for the alert. Next, you must add at least one sender and a recipient. We also recommend adding email groups if you want to notify more than a few people of an alert. You can configure senders and recipients using **Manage senders** and **Manage email groups**. 
+
+#### Manage senders
+Manage senders allows you to configure and manage **Sender** email addresses. Senders are email accounts from where the alert notification is sent to different recipients or email groups. 
+
+To configure a sender email, do the following:
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**, **New sender**. Multiple senders can be added one at a time.
+1. Enter a unique **Sender name**. Duplicates are not allowed.
+1. Enter the **Email address**, SMTP **Host** (e.g. `smtp.gmail.com` for a Gmail account), and the **Port** number.
+1. You can choose to use an **Encryption method** or leave it as **None**. **SSL** or **TLS** are standard protocols to secure your email. To use SSL or TLS encryption, you must enter your secure credentials to verify access to the account i.e. your username and password. You can enter these credentials in the Elasticsearch Keystore using the CLI. Run the following commands (in the bin directory of your Elasticsearch directory) to enter your username and password. The `<sender_name>` is the name you entered for **Sender**.
```

Comment:
It might be worth noting that depending on the email that they're using to send with, **SSL** or **TLS** might be required. Since essentially all normal emails are password protected, this is usually the case.

Here is a helpful post for [using Gmail's SMTP server](https://support.google.com/a/answer/176600).
Specifically the section that says "Use the Gmail SMTP server".

Reply:
I've updated it like this - **SSL** or **TLS** are standard protocols to secure your email and send it using the SMTP host as your server. Using SSL or TLS requires you to authenticate with your user ID and password. You can enter these credentials in the Elasticsearch Keystore using the CLI. Run the following commands (in the bin directory of your Elasticsearch directory) to enter your username and password.